### PR TITLE
New host actions: enable/disable

### PR DESCRIPTION
**Pull request by mareklibra**

### CODE REVIEW

**Comment by mareklibra** [comment]
01_disable
---
02_disableError
---
04_disabled
---
05_enabled
---
06_enabling

File: src/components/clusterWizard/HostsTable.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.host-table-alerts {
+  padding-top: 0.5rem !important;
```
Comment (jtomasek) [inline src/components/clusterWizard/HostsTable.scss:2]: I think we should use variable
```suggestion
  padding-top: var(--pf-global--spacer--lg) !important;
```

Also the file does not have to be `scss`, we most likely won't need any scss features and can use plain css.

File: src/components/clusterWizard/HostsTable.tsx [hunk 2]
```diff
@@ -116,17 +131,90 @@ const HostsTable: React.FC<HostsTableProps> = ({ hosts = [], uiState, fetchHosts
     [hostRows, openRows],
   );
 
+  const addAlert = React.useCallback(
+    (alert: Alert) => {
+      if (alert.key) {
```
Comment (jtomasek) [inline src/components/clusterWizard/HostsTable.tsx:136]: When adding an alert with the same key which already exists, do we want to replace the alert?

File: src/components/clusterWizard/HostsTable.scss (hunk 1 above)
Comment (mareklibra) [reply src/components/clusterWizard/HostsTable.scss]: Renamed

File: src/components/clusterWizard/HostsTable.tsx (hunk 2 above)
Comment (mareklibra) [reply src/components/clusterWizard/HostsTable.tsx:136]: Good point. With recent implementation, the alert is not duplicated but moved to the top instead.

**Pull request merged**